config/config.go

```diff
@@ -124,9 +124,10 @@ func (e Endpoint) BuildEthereumOptions() []ethereum.Option {
 }
 
 type Database struct {
-	Driver    database.Driver `mapstructure:"driver" validate:"required" default:"postgres"`
-	Partition *bool           `mapstructure:"partition" validate:"required" default:"true"`
-	URI       string          `mapstructure:"uri" validate:"required" default:"postgres://postgres@localhost:5432/postgres"`
+	CoveragePeriod int             `mapstructure:"coverage_period" validate:"required" default:"3"`
```

Comment:
should also add coverage_period config in min/default config file.

and should also notify Explorer to modify the config file gereration logics.